### PR TITLE
[frontend] fix relationship stop_time edition (#4133)

### LINKED ISSUE #4133: Can't update relationship stop_time
Go to a relationship.
Edit the stop_time.
The update is not taken into account.

### PR DESCRIPTION
**Pull request by Archidoit**
Github issue: https://github.com/OpenCTI-Platform/opencti/issues/4133

### CODE REVIEW

File: opencti-platform/opencti-front/src/private/components/common/stix_core_relationships/StixCoreRelationshipEditionOverview.tsx [hunk 1]
```diff
@@ -275,6 +275,18 @@ Omit<StixCoreRelationshipEditionOverviewProps, 'queryRef'>
     stixCoreRelationshipValidator,
   );
 
+  // necessary for stop_time because the validator includes a reference to another value (start_time)
```
Comment (tomibennett) [inline opencti-platform/opencti-front/src/private/components/common/stix_core_relationships/StixCoreRelationshipEditionOverview.tsx:278]: If the validator refers to the wrong value, should it be fixed?

Comment (Archidoit) [reply opencti-platform/opencti-front/src/private/components/common/stix_core_relationships/StixCoreRelationshipEditionOverview.tsx:278]: it's not a wrong value, you need it to compare start_time and stop_time

File: opencti-platform/opencti-front/src/private/components/common/stix_core_relationships/StixCoreRelationshipEditionOverview.tsx [hunk 2]
```diff
@@ -383,7 +395,7 @@ Omit<StixCoreRelationshipEditionOverviewProps, 'queryRef'>
                 component={DateTimePickerField}
                 name="stop_time"
                 onFocus={editor.changeFocus}
-                onSubmit={editor.changeField}
+                onSubmit={handleSubmitFieldStopTime}
```
Comment (SouadHadjiat) [inline opencti-platform/opencti-front/src/private/components/common/stix_core_relationships/StixCoreRelationshipEditionOverview.tsx:398]: why doesn't it work like the other fields ? why did you need to add `handleSubmitFieldStopTime` ?

Comment (Archidoit) [reply opencti-platform/opencti-front/src/private/components/common/stix_core_relationships/StixCoreRelationshipEditionOverview.tsx:398]: the other fields don't call an other field for their validation

Comment (Archidoit) [reply opencti-platform/opencti-front/src/private/components/common/stix_core_relationships/StixCoreRelationshipEditionOverview.tsx:398]: the origin of the problem is the same as for an old issue whose brainstorm is here : https://www.notion.so/filigran/Can-t-update-end-date-of-an-event-3033-ae8c2ab25c564973b974dfb3c980c90d#3e295d32596d4da391830a004964244e

**Review by SarahBocognano — APPROVED** [review]
Tested ✅

**Pull request merged**